Tests/Tests.UWP/FodyWeavers.xml

```diff
@@ -1,3 +1,3 @@
 ﻿<Weavers xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="FodyWeavers.xsd">

```

Comment:
Can you refresh my memory on what this file is supposed to be for and why only the UWP tests have it?

Reply:
All tests have it, we must have forgotten to disable analytics on UWP which is why this is the only one that shows up as changed. The other tests pull the FodyWeavers.xml from the common Realm.Tests project because they're using the new csproj format, whereas the uwp project hasn't been upgraded or just doesn't support it.